doc/source/whatsnew/v1.1.1.rst

```diff
@@ -50,6 +50,10 @@ Categorical
 
 -
 
+**Other**
```

Comment:
can you put in 1.2? (also don't put in other generally)

Reply:
or is this a regression / easy patch?

Reply:
i dont think its a regression, but it is a bugfix

Reply:
ok its in 1.1.1 that's ok